File: vortexdbg-android/src/main/java/com/vortexdbg/linux/android/dvm/api/AssetManager.java
package com.vortexdbg.linux.android.dvm.api;

import com.vortexdbg.linux.android.dvm.DvmObject;
import com.vortexdbg.linux.android.dvm.VM;

public class AssetManager extends DvmObject<String> {

    public AssetManager(VM vm, String value) {
        super(vm.resolveClass("android/content/res/AssetManager"), value);
    }

}

File: vortexdbg-android/src/main/kotlin/com/vortexdbg/linux/android/dvm/api/AssetManager.kt
package com.vortexdbg.linux.android.dvm.api

import com.vortexdbg.linux.android.dvm.DvmObject
import com.vortexdbg.linux.android.dvm.VM

open class AssetManager(vm: VM, value: String) : DvmObject<String>(vm.resolveClass("android/content/res/AssetManager"), value)
